docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+
+services:
+  book-store:
```

Comment:
I'd recommend you change the order. Better to start services and then service that depends on others.